package.json

```diff
@@ -12,11 +12,11 @@
     "test:watch": "jest --watch --verbose",
     "test:coverage": "jest --coverage --colors",
     "prepare-folder": "cp -r ./ ../U-confort-dist && rm -rf ../U-confort-dist/.git && rm -rf ../U-confort-dist/node_modules",
-    "zip": "npm run build && npm run prepare-folder && zip -r ../U-confort.zip ../U-confort"
+    "zip": "npm run build && npm run prepare-folder && zip -r ../U-confort.zip ../U-confort-dist"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "npm test && lint-staged"
```

Comment:
any reason to run the tests under `pre-commit` script?

Reply:
If the test fails, we shouldn't make a commit.